app/models/candidate.rb

```diff
@@ -4,15 +4,18 @@ class Candidate < ActiveRecord::Base
   belongs_to :profile
   belongs_to :post
 
-  validates :profile_id, uniqueness: { scope: [:post_id,:election_id],message: "har redan en likadan kandidatur"},  on: :create
-  validates :name,:lastname,:stil_id,:email,:phone, :post,:profile,:election, presence: true, on: :create
-  validates :name,:lastname,:stil_id,:email,:phone, :profile,:election, presence: true, on: :update
+  validates :profile_id, uniqueness: {
+                           scope: [:post_id, :election_id],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.